.github/workflows/__delete.yml

```diff
@@ -0,0 +1,24 @@
+on:
+  workflow_call:
+    inputs:
+      gotenberg_version:
+        type: string
+        required: true
+      alternate_repository:
+        type: string
+        default: ''
+
+permissions:
+  contents: read
+
+jobs:
+  delete:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Delete
+        uses: gotenberg/gotenberg-release-action/actions/delete@main
```

Comment:
@gulien is the repo with this action not public?

Reply:
nevermind,i see they are local now

Reply:
@bendavies hat brought you here? 😄